Lib/fontTools/cffLib/specializer.py

```diff
@@ -220,14 +223,15 @@ def generalizeCommands(commands, ignoreErrors=True):
 			if ignoreErrors:
 				# Store op as data, such that consumers of commands do not have to
 				# deal with incorrect number of arguments.
-				result.append(('', args))
-				result.append(('', [op]))
+				if len(args):
```

Comment:
> That was intentional.

I'm aware of that. But I'm not sure how some test cases can pass without it. I should have just committed the changes to the tests and let you deal with the failures 🤣